07-bootstrap-validation.md

```diff
@@ -26,6 +26,42 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
+
+   > The node update phase of the cluster lifecycle belongs to day2 operations. Cluster ops will update their node images as regular as required for two main reasons, the first one is for the Kubernetes cluster version, and the second one is to keep up with node-level OS updates. A new AKS release will be introducing new features such as new addons as well as making new kubernetes versions available while new AKS Node Images introduce changes at the OS level. Both release types follow Azure Safe Deployments Practices to roll out to all regions. For more information please take a look [How to use the release tracker](https://learn.microsoft.com/azure/aks/release-tracker#how-to-use-the-release-tracker). Additionally, cluster ops want to keep up with supported kubernetes versions for SLA reasons as well as to prevent from piled up updates since version updates can't be skipped at one's discretion. For more information, please take a look at [Kubernetes version upgrades](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#kubernetes-version-upgrades).
+
+   > Once a new update is available, it can be applied manually for the greatest degree of control by placing requests against the Azure control plane. Alternatevely, ops team could opt-in to automatically update to the latest available version by configuring an udpate channel following a desired cadence combining this with a planned maintenance window, one for kubernetes version updates and another one for OS level upgrades. AKS provides with two configurable different auto-upgrade channels dedicated to the oforementioned update types. For more information, please refer to  [Upgrade options for Azure Kubernetes Service (AKS) clusters](https://learn.microsoft.com/azure/aks/upgrade-cluster). Nodepools in this AKS cluster span into multiple availability zones, so an important consideration is that automatic updates are conducted based on a best-effort zone balancing in node groups. Nodes Max Surge and Pod Disruption Budget are configured in this baseline to prevent from unbalanced zones increasing the Availabilty. By default clusters nodes are updated one at the time. Max Surge has the ability to increase or reduce the speed of a cluster upgrade. In clusters with 6+ nodes hosting worloads that are sensitive to disruptions, it is recommended up to `33%` surge to achieve a safe upgrade pace. For more information, please take a look at [Customer node surge upgrade](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#customize-node-surge-upgrade). To prevent from disruption, production clusters should be configured with [node draining timeout](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-drain-timeout-valuei) and [soak time](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-soak-time-value) by taking into account the specific charactistics of their workloads.
```

Comment:
```suggestion
   > When a new update becomes available, it can be manually applied for maximum control by making requests against the Azure control plane. Alternatively, the operations team can opt to automatically update to the latest version by configuring an update channel to follow the desired cadence. This can be combined with a planned maintenance window, one for Kubernetes version updates and another for OS-level upgrades. AKS offers two different configurable auto-upgrade channels dedicated to these update types. For more information, please refer to [Upgrade options for Azure Kubernetes Service (AKS) clusters](https://learn.microsoft.com/azure/aks/upgrade-cluster).
> Node pools in this AKS cluster span multiple availability zones. Therefore, it’s important to note that automatic updates are conducted based on a best-effort zone balancing in node groups. To prevent zone imbalance and increase availability, Nodes Max Surge and Pod Disruption Budget are configured in this baseline. By default, cluster nodes are updated one at a time. Max Surge can adjust the speed of a cluster upgrade. In clusters with 6+ nodes hosting disruption-sensitive workloads, a surge of up to `33%` is recommended for a safe upgrade pace. For more information, please see [Customer node surge upgrade](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#customize-node-surge-upgrade). To minimize disruption, production clusters should be configured with [node draining timeout](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-drain-timeout-valuei) and [soak time](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-soak-time-value), taking into account the specific characteristics of their workloads.
```